azure-pipelines/build.yml

```diff
@@ -110,6 +110,15 @@ steps:
     arguments: '-target=dotnetpackpublished -mono'
   continueOnError: true
 
+- task: BatchScript@1
+  displayName: "Package service tool projects"
+  env:
+    BUILD_DOTNET_TOOL: "true"
+  inputs:
+    filename: build.cmd
+    arguments: "-target=dotnetpackservicetools -mono"
+  continueOnError: true
```

Comment:
I'd remove this - there shouldn't be any reason we'd expect it to fail unless I'm missing something

Reply:
We do the same thing with the existing publish task above this one.

Reply:
And I'd suggest that we remove that too, except we'll get rid of that step as a follow up item anyways. We want this to be generated for every build so I'd rather the whole build fail if something breaks (which given that it's just building stuff locally seems...unlikely...to begin with)